r/R/arrow-package.R

```diff
@@ -65,6 +65,14 @@
     # Disable multithreading on Windows
     # See https://issues.apache.org/jira/browse/ARROW-8379
     options(arrow.use_threads = FALSE)
+
+    # Try to set timezone database
+    if (requireNamespace("tzdb", quietly = TRUE)) {
+      tzdb::tzdb_initialize()
+      set_timezone_database(tzdb::tzdb_path("text"))
+    } else {
+      warning("tzdb not installed. Timezones will not be available.")
```

Comment:
```suggestion
      warning("The tzdb package is not installed. Timezones will not be available.")
```

Just a suggestion on the wording here, since tzdb is a fairly hidden and unknown package (purposefully)

Reply:
Thanks!